R/prep_data.R

```diff
@@ -139,17 +162,18 @@ prep_data <- function(d,
                       convert_dates = TRUE,
                       impute = TRUE,
                       collapse_rare_factors = TRUE,
-                      center = FALSE,
-                      scale = FALSE,
+                      PCA = FALSE,
+                      center = PCA,
+                      scale = PCA,
                       make_dummies = TRUE,
                       add_levels = TRUE,
                       logical_to_numeric = TRUE,
                       factor_outcome = TRUE,
                       no_prep = FALSE) {
   # Check to make sure that d is a dframe
-  if (!is.data.frame(d))
+  if (!is.data.frame(d)) {
     stop("\"d\" must be a data frame.")
-
+  }
```

Comment:
No brackets here. We've been leaving them off when the code block within the `if` is only one line.